Monika After Story/game/script-topics.rpy

```diff
@@ -4791,6 +4791,33 @@ label monika_urgent:
     m 1k "I'm sure it's adorable~"
     return
 
+init 5 python:
+    for key in ['homework','assignment']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_homework')
+    monika_random_topics.append('monika_homework')
+
+label monika_homework:
+      m 1a "Say, [player]..."
+      m 2d "Are you having trouble with your assignments?"
+      menu:
+        "Yes.":
+            m 1h "Hmm.."
+            m "You should try completing them one by one, it's much easier that way."
+            m 1b "Proper organization and following a schedule can also help a lot in easing the workload."
+            m "Aside from that, getting rid of distractions could help in finishing the task much quicker."
+            m "I also suggest finishing tasks as early as possible, as cramming them last minute can be quite problematic."
+            m 1a"I hope that could've been of some use to you."
+            m 2g "I don't want my sweetheart to be struggling in school or with work, okay?"
+            m 1b "I'm sure you'll be just fine, ahaha~"
+        "No.":
+            m 1a "I'm glad to hear that then."
+            m 1d "Some people have a hard time dealing with work and it starts to pile up gradually."
+            m 1a "So keep up the good work, [player]. Don't stress yourself out too much."
+            m "I believe in you~"
+            return
```

Comment:
'return' needs to be indented correctly. Refer to other topics to see correct way.